resources/views/settings/subscription/index.blade.php

```diff
@@ -6,6 +6,17 @@
  */
 
 ?>
+@php
+    $country = 'CH';
```

Comment:
Please put this in the controller, or better yet in a service, so we can re-use it on the other subscriptions page :)